test_src/sljitTest.c

```diff
@@ -11647,15 +11650,15 @@ static void test92(void)
 	sljit_emit_atomic_load(compiler, SLJIT_MOV32, SLJIT_R0, SLJIT_R2);
 	sljit_emit_op1(compiler, SLJIT_MOV32, SLJIT_S1, 0, SLJIT_R0, 0);
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_S2, 0, SLJIT_R0, 0);
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, -573621);
+	sljit_emit_op1(compiler, SLJIT_MOV32, SLJIT_R0, 0, SLJIT_IMM, -573621);
```

Comment:
This way you don't test that the upper 32 bits on 64 bit are preserved.

Reply:
I made that test explicit with case 27, and to protect against false positives when the filling was 0xff, but you are correct it is no longer needed; will remove